app/src/pages/settings/SettingsPage.tsx

```diff
@@ -49,75 +55,99 @@ export function SettingsPage() {
   return (
     <main css={settingsPageCSS}>
       <div css={settingsPageInnerCSS}>
-        <Flex direction="column" gap="size-200" width="100%">
-          <Flex direction="row" gap="size-200" alignItems="center">
-            <View flex="2">
-              <Card title="Platform Settings" variant="compact">
-                <form css={formCSS}>
-                  <Flex direction="row" gap="size-100" alignItems="end">
-                    <TextField value={BASE_URL} isReadOnly>
-                      <Label>Hostname</Label>
-                      <Input />
-                      <Text slot="description">
-                        Connect to Phoenix over HTTP
-                      </Text>
-                    </TextField>
-                    <CopyToClipboardButtonWithPadding text={BASE_URL} />
-                  </Flex>
-                  <Flex direction="row" gap="size-100" alignItems="end">
-                    <TextField value={VERSION} isReadOnly>
-                      <Label>Platform Version</Label>
-                      <Input />
-                      <Text slot="description">
-                        The version of the Phoenix server
-                      </Text>
-                    </TextField>
-                    <CopyToClipboardButtonWithPadding text={VERSION} />
-                  </Flex>
-                  <Flex
-                    direction="row"
-                    gap="size-100"
-                    alignItems="end"
-                    justifyContent="center"
-                  >
-                    <TextField
-                      value={`pip install 'arize-phoenix==${VERSION}'`}
-                      isReadOnly
-                    >
-                      <Label>Python Version</Label>
-                      <Input />
-                      <Text slot="description">
-                        The version of the Python client library to use to
-                        connect to this Phoenix
-                      </Text>
-                    </TextField>
-                    <CopyToClipboardButtonWithPadding text={VERSION} />
-                  </Flex>
-                </form>
-              </Card>
-            </View>
-            <View flex="1">
-              <Card title="Database Usage" variant="compact">
-                <DBUsagePieChart query={data} />
-              </Card>
-            </View>
-          </Flex>
-          <IsAdmin>
-            <>
-              <APIKeysCard />
-              <UsersCard />
-            </>
-          </IsAdmin>
-          <GenerativeProvidersCard query={data} />
-        </Flex>
+        <Tabs>
+          <TabList>
+            <Tab id="general">General</Tab>
+            <Tab id="ai-providers">AI Providers</Tab>
+            <Tab id="annotations">Annotations</Tab>
```

Comment:
should we add routes for these? Might be good so we can link to providers directly.